src/config.ts

```diff
@@ -12,19 +12,37 @@ import { pipe } from "fp-ts/lib/function";
 
 import * as reporters from "@pagopa/ts-commons/lib/reporters";
 import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { UrlFromString } from "@pagopa/ts-commons/lib/url";
+import { withDefault } from "@pagopa/ts-commons/lib/types";
+import { NumberFromString } from "@pagopa/ts-commons/lib/numbers";
 
 // ----------------------------
 // Global app configuration
 // ----------------------------
+const GetAssertionConfig = t.type(
+  {
+    LOLLIPOP_GET_ASSERTION_API_KEY: NonEmptyString,
+    LOLLIPOP_GET_ASSERTION_BASE_URL: UrlFromString
```

Comment:
Names are different from the ones defined on [infra](https://github.com/pagopa/io-infra/blob/d664d473ffdd4cee8dfe482b2923786c8a9f4f69/src/domains/citizen-auth-app/07_function_fast_login.tf#L26)

Reply:
Yes, IMHO should be changed in infra. `LC` prefix refers to the LollipopConsumer but in here is this function the LC and the parameters refers to the fn-lollipop.

Reply:
https://github.com/pagopa/io-infra/pull/600